dtfs-central-api/src/v1/controllers/portal/utilisation-report-service/previous-reports.controller.js

```diff
@@ -17,11 +17,11 @@ const getUtilisationReports = async (req, res) => {
         },
       },
     ]).toArray();
-    return res.status(200).send(filteredAndSortedUtilisationReports);
+    res.status(200).send(filteredAndSortedUtilisationReports);
   } catch (error) {
     console.error('Unable to get utilisation reports %s', error);
+    res.status(500).send({ status: 500, message: error });
```

Comment:
As mentioned in person - can we change returning the code error to a generic 'Failed to get utilisation reports' style string